src/frontend/Dockerfile

```diff
@@ -8,15 +8,25 @@ COPY package*.json ./
 FROM base as dev
 ENV NODE_ENV=development
 ENV BUILD_PATH=build
-RUN npm ci --verbose --production=false && npm cache clean --force
+RUN npm i --verbose
 RUN apt-get update && apt-get install make
 RUN npm install -g serve
 USER node
 COPY . .
 ENTRYPOINT ["./entrypoint.sh"]
 
+# Actual prod image used for deployments
+FROM base as prod
+ENV NODE_ENV=production
```

Comment:
Actual prod build image will run `npm ci` with production flag via `NODE_ENV`, which will only install prod deps instead of dev deps!